### PR TITLE
remove dead code (scripts/release.py)

### PR DESCRIPTION
**Pull request by prometheanfire**
Signed-off-by: Matthew Thode <mthode@mthode.org>

### CODE REVIEW

**Comment by odyssey4me** [comment]
Tox is failing due to a failed test. Looks like that will need adjustment of tox config, maybe gating config and another script removed.

**Comment by prometheanfire** [comment]
recheck_cit_xenial_swift_deploy

File: tox.ini [hunk 1]
```diff
@@ -50,9 +50,6 @@ commands =
         ansible-galaxy install -r {homedir}/openstack-ansible-role-requirements.yml; \
         {toxinidir}/scripts/linting-ansible.sh"
 
-[testenv:release-script]
-commands = python -m unittest discover -s scripts -p test_release.py -v
```
Comment (odyssey4me) [inline tox.ini]: Given that test_release.py is no longer used, it should be removed

Comment (prometheanfire) [reply tox.ini]: addressed

**Pull request merged**